src/Data/Vec/Properties.agda

```diff
@@ -215,12 +172,8 @@ lookup⇒[]= (suc i) (_ ∷ xs) p    = there (lookup⇒[]= i xs p)
 
 lookup-inject≤-take : ∀ m (m≤m+n : m ≤ m + n) (i : Fin m) (xs : Vec A (m + n)) →
                       lookup xs (Fin.inject≤ i m≤m+n) ≡ lookup (take m xs) i
-lookup-inject≤-take (suc m) m≤m+n zero (x ∷ xs)
-  rewrite unfold-take m x xs = refl
-lookup-inject≤-take (suc (suc m)) (s≤s m≤m+n) (suc zero) (x ∷ y ∷ xs)
-  rewrite unfold-take (suc m) x (y ∷ xs) | unfold-take m y xs = refl
-lookup-inject≤-take (suc (suc m)) (s≤s (s≤s m≤m+n)) (suc (suc i)) (x ∷ y ∷ xs)
-  rewrite unfold-take (suc m) x (y ∷ xs) | unfold-take m y xs = lookup-inject≤-take m m≤m+n i xs
+lookup-inject≤-take (suc m) m≤m+n zero (x ∷ xs) = refl
+lookup-inject≤-take (suc m) (s≤s m≤m+n) (suc i) (x ∷ xs) = lookup-inject≤-take m m≤m+n i xs
```

Comment:
In the spirit of many prior PRs (and open ones eg #2000) aiming to make computation with (irrelevant) proofs of `_≤_` as lazy as possible, I'd be tempted to write this second clause as 
```agda
lookup-inject≤-take (suc m) le (suc i) (x ∷ xs) = lookup-inject≤-take m (≤-pred le) i xs
```

Reply:
Oops: computer says no! At least not until the operations on `Fin` become lazier again... (more sackcloth and ashes for me, I'm afraid)

Reply:
Will 'resolve' this one for now, and return to it in a subsequent PR...